shuttle/templates/schedules/load_logs.html

```diff
@@ -0,0 +1,102 @@
+{% block body_content %}
+    {% set ns = namespace(first_time_shuttle = true) %}
+    {% for i in range(logs|length) %}
+        {% if logs[i]['log_date'] == date and logs[i]['location'] != None %}
```

Comment:
Instead of making sure you have to right date in html using jinja, you could probably just do it in python and then just pass over the data. Once `shuttle_logs` def gets the selected date could probably just run a query where you only get results with the matching date.